AWS/ecs-fargate/main.tf

```diff
@@ -523,10 +556,35 @@ resource "aws_ecs_task_definition" "dbeaver_rm" {
     }
   }
   
-  container_definitions = jsonencode([{
+  container_definitions = jsonencode([
+  {
+    name      = "${var.deployment_id}-init-rm-permissions"
+    image     = "busybox:latest"
+    essential = false
+    command   = ["sh", "-c", "mkdir -p /workspace/.metadata && chown -R 8978:8978 /workspace && chmod -R 755 /workspace"]
```

Comment:
append failed state logging:
```command && command || echo "failed: cant chown```
use list style for commands
[
"1",
"2",
...
]